NuEVI/menu.cpp

```diff
@@ -501,11 +501,22 @@ const MenuEntrySub specialKeyMenu = {
   , nullptr
 };
 
+
+const MenuEntrySub wlPowerMenu = {
+  MenuType::ESub, "WL POWER", "WL POWER", &wlPower, 0, 3, MenuEntryFlags::ENone,
+  [](SubMenuRef __unused, char* out, const char** __unused unit) {
```

Comment:
We should add a task to change the lambda usage to normal function pointers. For some reason the lambdas uses more ram.